### PR TITLE
Aws stuff

### PR DESCRIPTION
**Pull request by Geor23**

### CODE REVIEW

**Comment by susanthomson** [comment]
A rebase and squash maybe?

**Comment by Geor23** [comment]
does it really matter? I just really need this quickly to set up autodeploy and cirlce ci onto stephen's repo

**Comment by susanthomson** [comment]
I think it does matter because it's going straight into master and it's going to make the tree super messy for everyone. Also it's making changes to css for some reason which I think the rebase would fix.

File: client/css/main.css [hunk 1]
```diff
@@ -7,6 +7,25 @@ html::-webkit-scrollbar {
...
+.banner-container {
+	width: 91%; 
+	margin: 30px; 
+	background-color: black;
+}
+
+.banner {
+	width: 100%;
+}
+
```
Comment (BRHoward) [inline client/css/main.css:30]: dont think this css is used?

**Comment by BRHoward** [comment]
also probs doesn't matter for this but try not to push straight to master :jamaica:

**Pull request closed without merging**